automatic/ruby.install/tools/chocolateyInstall.ps1

```diff
@@ -12,7 +12,7 @@ if ( !$pp['NoPath'] ) { $tasks += 'modpath'  }
 Write-Host "Ruby is going to be installed in '$installDir'"
 
 $packageArgs = @{
-  packageName    = 'ruby'
+  packageName    = 'ruby.install'
   fileType       = 'exe'
   file           = Get-Item "$toolsDir\*_x32.exe"
   file64         = Get-Item "$toolsDir\*_x64.exe"
```

Comment:
I know you did not change any lines here, but could you update these two lines to say:

```suggestion
  file           = "$toolsDir\"
  file64         = "$toolsDir\
```